pyro/distributions/__init__.py

```diff
@@ -19,15 +22,13 @@
 from pyro.distributions.torch import __all__ as torch_dists
 from pyro.distributions.torch_distribution import TorchDistribution
 from pyro.distributions.torch_transform import TransformModule
-from pyro.distributions.conditional import (ConditionalDistribution, ConditionalTransform, ConditionalTransformModule,
-                                            ConditionalTransformedDistribution)
-from pyro.distributions.util import enable_validation, is_validation_enabled, validation_enabled
+from pyro.distributions.transforms import *  # noqa F403
```

Comment:
Won't this also import all torch transforms into the distributions namespace? Are there any reasons for doing so?

Reply:
Yes, you're right. We already do this, and I had avoided making this change in this PR, but I guess we can do this now.